service/src/main/java/com/commercetools/pspadapter/payone/mapping/PayoneRequestFactory.java

```diff
@@ -33,4 +36,31 @@ public CaptureRequest createCaptureRequest(final PaymentWithCartLike paymentWith
         throw new UnsupportedOperationException("this request type is not supported by this payment method.");
     }
 
+    protected void mapFormPaymentWithCartLike(final AuthorizationRequest request,
+                                              final PaymentWithCartLike paymentWithCartLike,
+                                              final Logger logger) {
+        final Payment ctPayment = paymentWithCartLike.getPayment();
+        final CartLike ctCartLike = paymentWithCartLike.getCartLike();
+
+        MappingUtil.mapCustomFieldsFromPayment(request, ctPayment.getCustom());
+
+
+        try {
+            MappingUtil.mapCustomerToRequest(request, ctPayment.getCustomer());
+        } catch (final IllegalArgumentException ex) {
+            logger.warn("Could not fully map payment with ID " + paymentWithCartLike.getPayment().getId(), ex.getMessage());
+        }
+
+        try {
+            MappingUtil.mapBillingAddressToRequest(request, ctCartLike.getBillingAddress());
+        } catch (final IllegalArgumentException ex) {
+            logger.warn("Could not fully map payment with ID " + paymentWithCartLike.getPayment().getId(), ex.getMessage());
+        }
+
+        try {
+            MappingUtil.mapShippingAddressToRequest(request, ctCartLike.getShippingAddress());
+        } catch (final IllegalArgumentException ex) {
+            logger.warn("Could not fully map payment with ID " + paymentWithCartLike.getPayment().getId(), ex.getMessage());
+        }
```

Comment:
1. Looks like may be `static`

2. Why only `mapBillingAddressToRequest` logged as an _error_, while all other as _debug_?